conda/cli/activate.py

```diff
@@ -62,7 +81,7 @@ def main():
             print(os.environ['PATH'])
             raise
         paths = []
-        sys.stderr.write("discarding %s from PATH\n" % binpath)
+        sys.stderr.write("discarding %s from PATH\n" % pathlist_to_str(binpath))
```

Comment:
I see your point. What kind of shortcuts do you have in mind?

So far, I thought that putting `<conda_root>\Scripts` in the PATH to begin with would be the standard setup on Windows. Maybe you should do it like `git` who created another subdirectory (say, `<conda_root>\cmd`) that only contains `activate.bat`, `deactivate.bat` and the conda scripts, and that can be safely added to and kept in the PATH? Alternatively, conda could remember if `<conda_root>\Scripts` was already in the PATH and act accordingly.